src/components/NavBar/NavDropdown.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box } from 'nft/components/Box'
+import { ReactNode } from 'react'
+
+import * as styles from './NavDropdown.css'
+
+interface NavDropdownProps {
+  top: number
+  right?: number
+  leftAligned?: boolean
+  horizontalPadding?: boolean
+  centerHorizontally?: boolean
+  children: ReactNode
+}
+
+export const NavDropdown = ({
+  top,
+  centerHorizontally,
+  leftAligned,
+  horizontalPadding,
+  children,
+}: NavDropdownProps) => {
+  return (
+    <Box
+      paddingX={horizontalPadding ? '16' : undefined}
```

Comment:
Why isn't this in the `.css.ts` file?

Reply:
Because as a style choice we don't pass variables, ie `horizontalPadding` in this case, into the css files. This way logic is kept in the tsx files.